### ISSUE TITLE
Messages

### ISSUE THREAD

**Issue by astrofrog** [astrofrog/test-bot#6]
Preview of current messages

**Comment by astrofrog** [comment]
Hi humans :wave: - this issue was labeled as **Close?** approximately 5 months ago. So..... any news? :newspaper_roll:

If you think this issue should not be closed, a maintainer should remove the **Close?** label - otherwise, I'm just gonna have to close this issue in 30 days. Your time starts now! Tick tock :clock10:

*If you believe I commented on this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues)*

**Comment by astrofrog** [comment]
:alarm_clock: Time's up! :alarm_clock:

I'm going to close this issue as per my previous message. But if you feel that we should really really keep this open, then feel free to re-open and remove the **Close?** label. But no one has done anything for 6 months, so... Just sayin!

*If this is the first time I am commenting on this issue, or if you believe I closed this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues)*

**Comment by astrofrog** [comment]
Hi humans :wave: - this pull request hasn't had any new commits for approximately {pasttime}. Of course, in this day and age we all have way too much on our plates (especially me!) but in the interest on making sure that we don't keep pull requests open if they are no longer relevant, **I plan to close this in {futuretime} if the pull request doesn't have any new commits by then.**

If you **really** want to keep this pull request open beyond because it needs more discussion, then you can get a maintainer to add the **keep-open** label, but please only use this in rare cases. A better solution if you don't agree to merge this now is to close this and open a new issue to remind ourselves this should be done (for example if this pull request is the wrong approach).

In any case, I will close this within a month unless there is a new commit or the **keep-open** label has been added. Thanks!

*If you believe I commented on this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues)*

**Comment by astrofrog** [comment]
:alarm_clock: Time's up! :alarm_clock:

I'm going to close this pull request as per my previous message. If you think what is being added/fixed here is still important, please remember to open an issue to keep track of it. Thanks!

*If this is the first time I am commenting on this issue, or if you believe I closed this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues)*

**Comment by pllim** [comment]
Shorter version of https://github.com/astrofrog/test-bot/issues/6#issuecomment-332195362 
---------------------
Hi humans 👋 - this pull request hasn't had any new commits for approximately {pasttime}. **I plan to close this in {futuretime} if the pull request doesn't have any new commits by then.**

In lieu of a stalled pull request, please close this and open an issue instead to revisit in the future. Maintainer may also choose to add `keep-open` label to keep this PR open but it is discouraged unless absolutely necessary.

*If you believe I commented on this issue incorrectly, please report this [here].*

**Comment by bsipocz** [comment]
> ... A better solution if you don't agree to merge this now...

@astrofrog - this wording doesn't seem right, quite often stale PRs are stale as the person opened them are abandoned them rather than addressed the review comment (e.g. most gsoc one), so from their point of those could be merged, but from the package point of view it's not true.

**Comment by astrofrog** [comment]
@bsipocz - do you like @pllim's wording?

**Comment by bsipocz** [comment]
Yes, I feel it's less ambiguous about the process, though could use a few more articles.

**Comment by astrofrog** [comment]
It's now:

Hi humans :wave: - this pull request hasn't had any new commits for approximately {pasttime}. **I plan to close this in {futuretime} if the pull request doesn't have any new commits by then.**

In lieu of a stalled pull request, please close this and open an issue instead to revisit in the future. Maintainers may also choose to add `keep-open` label to keep this PR open but it is discouraged unless absolutely necessary.

*If you believe I commented on this issue incorrectly, please report this [here](https://github.com/astropy/astropy-bot/issues).*

Looks ok?

**Comment by bsipocz** [comment]
Probably we don't need an issue every time, right?

> please close this and open an issue instead to revisit in the future.

please close this and open an issue instead if a reminder is needed to revisit in the future.

**Comment by bsipocz** [comment]
OK, so running this the first time on a repo, I think changing "approximately 5 months" to "at least 5 months" may make some sense?

**Comment by astrofrog** [comment]
Yes true - could you do a PR to the bot repo to implement these fixes?
